### PR TITLE
TEST/EXAMPLES: Update the documentation in ucp_client_server.c

### PR DESCRIPTION
**Pull request by alinask**

### CODE REVIEW

File: test/examples/ucp_client_server.c [hunk 1]
```diff
@@ -19,20 +19,12 @@
  * Notes:
  *
  *    - The server will listen to incoming connection requests on INADDR_ANY.
- *    - The client needs to pass the IP address of the server side to connect to,
- *      as the first and only argument to the test.
+ *    - The client needs to pass the IP address of the server side to connect to
```
Comment (hoopoepg) [inline test/examples/ucp_client_server.c]: not sure, but last "to" at end of this line should be removed as well

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7160/ for details (Mellanox internal link).

**Comment by alinask** [comment]
@yosefe please take a look.

**Comment by alinask** [comment]
@yosefe Can you please merge? i'll cherry-pick to v1.4

**Pull request merged**